prov/rxm/src/rxm_ep.c

```diff
@@ -2473,6 +2473,9 @@ int rxm_endpoint(struct fid_domain *domain, struct fi_info *info,
 	if (fi_param_get_int(&rxm_prov, "comp_per_progress",
 			     (int *)&rxm_ep->comp_per_progress))
 		rxm_ep->comp_per_progress = 1;
+	if (fi_param_get_int(&rxm_prov, "cm_progress_interval",
+				(int *) &rxm_cm_progress_interval))
+		rxm_cm_progress_interval = 10000;
```

Comment:
This initialization can be moved to RXM_INI since this is global.

Reply:
Done.